plugins/orchestrator-common/package.json

```diff
@@ -35,7 +35,8 @@
     "clean": "backstage-cli package clean",
     "prepack": "backstage-cli package prepack",
     "postpack": "backstage-cli package postpack",
-    "openapi": "./scripts/openapi.sh"
+    "openapi": "./scripts/openapi.sh",
+    "openapi:prettier:fix": "prettier --ignore-unknown --write ./src/auto-generated/api/models/schema.ts ./src/auto-generated/api/definition.ts ./src/auto-generated/docs/index.adoc/index.adoc"
```

Comment:
Is it possible to run it for the entire `./src/auto-generated` folder? After all, files in there are auto-generated. This way, we don't need to specify each file.

Reply:
sure! Done